game/scripts/npc/abilities/visage_summon_familiars_oaa.txt

```diff
@@ -56,7 +56,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "familiar_attack_damage"                          "25 50 75 150 300"
+        "familiar_attack_damage"                          "40 80 120 200 360" //oaa
```

Comment:
Familiar damage isn't the issue afaik but it can't hurt. //oaa not needed for custom ability.